test/test_pysodium.py

```diff
@@ -101,6 +101,16 @@ def test_aead_chacha20poly1305(self):
         output = pysodium.crypto_aead_chacha20poly1305_decrypt(output, ad, nonce, key)
         self.assertEqual(output, input_)
 
+    def test_aead_chacha20poly1305_ietf(self):
+        key = binascii.unhexlify(b"4290bcb154173531f314af57f3be3b5006da371ece272afa1b5dbdd1100a1007")
+        input_ = binascii.unhexlify(b"86d09974840bded2a5ca")
+        nonce = binascii.unhexlify(b"cd7cf67be39c794a")
+        ad = binascii.unhexlify(b"87e229d4500845a079c0")
+        output = pysodium.crypto_aead_chacha20poly1305_ietf_encrypt(input_, ad, nonce, key)
+        self.assertEqual(binascii.unhexlify(b"e3e446f7ede9a19b62a4677dabf4e3d24b876bb284753896e1d6"), output)
+        output = pysodium.crypto_aead_chacha20poly1305_ietf_decrypt(output, ad, nonce, key)
+        self.assertEqual(output, input_)
+
```

Comment:
how did you generate the test vectors? when i run the tests i get even a fail on one of them in line 110, in test_aead_chacha20poly1305_ietf:

    self.assertEqual(binascii.unhexlify(b"e3e446f7ede9a19b62a4677dabf4e3d24b876bb284753896e1d6"), output)
AssertionError: '\xe3\xe4F\xf7\xed\xe9\xa1\x9bb\xa4g}\xab\xf4\xe3\xd2K\x87k\xb2\x84u8\x96\xe1\xd6' != 'l\xa8\x03.\xd2\xd2\x8ae\xdb\x01\xf3\x9aW\xb9\xa9\xa8]n\x9a#\x1dF\xba\xc0\xdet'